docs/source/user_guide/users.ipynb

```diff
@@ -0,0 +1,1591 @@
+{
```

Comment:
<p>Just a proposal.. I think other notebooks tend to include an executable script to install required environment. Create a code cell and write:</p><pre class="ql-syntax" spellcheck="false">!conda create -n rapids-22.06 -c rapidsai -c nvidia -c conda-forge  \ 
    cuspatial=22.06 python=3.9 cudatoolkit=11.5 
</pre><p> </p><p>As for docker image, I'm not sure if we need to include it. Perhaps replace it with a line:</p><p><code>For other options to create RAPIDS environment, such as docker or build from source, see  <a href="..." target="_blank">RA</a>PIDS Release Selector</code></p><p> </p><p>How do we plan to track the version of these installation scripts? I've never tried to use <code>sed</code> to update a notebook before. Alternative is to include the conda script in a small breakout file <code>install_rapids_conda.sh</code> then do</p><pre class="ql-syntax" spellcheck="false">! chmod +x install_rapids_conda.sh &amp;&amp; ./install_rapids_conda.sh  
</pre><p>Then we can use https://github.com/rapidsai/cuspatial/blob/47ba833a139466028c8a756cc865b7c76d8ec586/ci/release/update-version.sh#L30-L31 to update <code>install_rapids_conda.sh</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuspatial/pull/666/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Before we add a file as a dependency for the user guide, do you mind if we try to use the script or a small modification of it to update the `.ipynb` directly? It should work as it is just another text file.

Reply:
Yes this is fine.